spark/v3.2/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestMetadataTables.java

```diff
@@ -319,6 +323,89 @@ public void testAllFilesPartitioned() throws Exception {
     TestHelpers.assertEqualsSafe(filesTableSchema.asStruct(), expectedFiles, actualFiles);
   }
 
+  @Test
+  public void testMetadataLogMetatable() throws Exception {
+    // Create table and insert data
+    sql("CREATE TABLE %s (id bigint, data string) " +
+        "USING iceberg " +
+        "PARTITIONED BY (data) " +
+        "TBLPROPERTIES " +
+        "('format-version'='2')", tableName);
```

Comment:
nit, no need for format-version =2?

Reply:
wanted to have [increasing sequence number per snapshot](https://github.com/apache/iceberg/blob/2df94844280e72d12d6692934556736e2b203714/core/src/main/java/org/apache/iceberg/TableMetadata.java#L369-L371), which is only possible with v2 format hence added the same otherwise sequence number would have remained `0`.

Reply:
Ah got it.